pkg/ee/cluster-backup/resources/user-cluster/static/backuprepositories.velero.io.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Where do these CRDs come from? How are they kept up-to-date? For cert-manager and the Velero chart, we have scripts in `hack/` to fetch them. These however.. ?

A script could also add a comment to these files, like https://github.com/kubermatic/kubermatic/blob/main/charts/backup/velero/crd/backuprepositories.yaml#L1, then you don't need an extra exclusion rule on the boilerplate checker anymore and the next guy after you will know how to update the CRDs.